examples/mongodb.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+require '../vendor/autoload.php';
+
+use Doctrine\MongoDB\Connection;
+use Doctrine\ODM\MongoDB\Configuration;
+use Doctrine\ODM\MongoDB\DocumentManager;
+use Doctrine\ODM\MongoDB\Mapping\Driver\AnnotationDriver;
+
+use Conticket\Document;
+
+AnnotationDriver::registerAnnotationClasses();
+
+$config = new Configuration();
+$config->setProxyDir('../data/cache/proxy');
+$config->setProxyNamespace('Proxies');
+$config->setHydratorDir('../data/cache/hydrators');
+$config->setHydratorNamespace('Hydrators');
+$config->setMetadataDriverImpl(AnnotationDriver::create('../data/cache/classes'));
+
+$dm = DocumentManager::create(new Connection(), $config);
+
+$user = new Document\User();
+$user->setName("Abdala Cerqueira");
+$user->setEmail("organizacao@phpeste.net");
```

Comment:
Can be `new Document\User($name = "Abdala", $email = "organizacao@phpeste.net");`, preserve immutability